rhcos-packages.yaml

```diff
@@ -10,7 +10,7 @@ packages:
   # installer iso packages
   - coreos-installer coreos-installer-bootinfra
   - ignition
-  - redhat-release-coreos
+  - redhat-release-coreos-48.84-1.el8 # Pinned for 8.4 Beta switch
```

Comment:
Minor: could also just do `redhat-release-coreos-48.84` so that we don't have to touch this again even if we have to respin the package.